packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -102,6 +102,7 @@ export interface AngularCompilerPluginOptions {
   missingTranslation?: string;
   platform?: PLATFORM;
   nameLazyFiles?: boolean;
+  lazyModulesResource?: string;
```

Comment:
`string | string[]`?

Reply:
I'm unsure what the final semantics here should be. Should we allow several extra lazy module resources? Each would be a file containing a loader and it will have all the lazy modules as context.

Reply:
Changed it always be an array.